### DISCUSSION TITLE
Como usar o Jump para pular entre perguntas dependendo da resposta anteriormente.

### DISCUSSION THREAD

**Discussion by danielpetinga** [ditdot-dev/vue-flow-form#285] [General]
Galera,

Estou com um código e gostaria de compartilhar com vocês uma dúvida que estou enfrentando e não consigo resolver. Tenho o seguinte cenário: estou trabalhando em um cadastro de cliente e, dependendo do sexo, quero abordar o cliente com uma mensagem diferente. Por exemplo, se o sexo for feminino, quero que a mensagem seja específica para mulheres e, em seguida, o código pule para a parte de inserção do email, sem passar pela mensagem destinada aos homens.

Detalhes: No primeiro salto do código, dependendo da escolha entre homem ou mulher, consigo pular corretamente. No entanto, no segundo salto, não está funcionando corretamente. Existe algum motivo específico para isso acontecer? SEGUE O CÓDIGO:

`  new QuestionModel({
    id: 'sexo',
    tagline: "Encontrar a rotina de cuidados perfeita para o seu cabelo é essencial. Um cabelo bonito começa com um bom diagnóstico, vamos começar! 😉",
    title: 'Você é?',
    helpTextShow: false,
    type: QuestionType.MultiplePictureChoice,
    multiple: false,
    required: true,
    options: [
        new ChoiceOption({
            imageSrc: require('@/assets/imagens/mulher.webp'),
            imageAlt: 'mulher',
            label: 'Uma Mulher',
            value: 'F'
        }),
        new ChoiceOption({
            imageSrc: require('@/assets/imagens/homem.webp'),
            imageAlt: 'homem',
            label: 'Um Homem',
            value: 'M'
        })
    ],
    jump: {
        F: 'nome_feminino',
        M: 'nome_masculino'
    }
}),

new QuestionModel({
    id: 'nome_feminino',
    tagline: 'Nós gostaríamos de conhecê-la.',
    title: 'Qual é o seu nome?',
    type: QuestionType.Text,
    required: true,
    placeholder: 'Por favor, digite o seu nome aqui...',
    jump: {
        F: 'email',
        M: 'email' // Adicionando um salto para 'email' também quando o sexo é masculino
    }
}),

new QuestionModel({
    id: 'nome_masculino',
    tagline: 'Nós gostaríamos de conhecê-lo.',
    title: 'Qual é o seu nome?',
    type: QuestionType.Text,
    required: true,
    placeholder: 'Por favor, digite o seu nome aqui...',
}),

new QuestionModel({
    id: 'email',
    tagline: "Nice to meet you 😊, let's continue",
    title: 'Fornecer um exemplo de e-mail.',
    type: QuestionType.Email,
    required: true,
    placeholder: 'Comece a digitar aqui...'
}),`

**Comment by spinn** [comment]
You can accomplish this by defining questions as components, documentation here:

https://www.ditdot.hr/en/docs/vue-flow-form-guide#question-components

I'd also recommend to post any future questions in English.

**Reply by danielpetinga** [reply]
OK thanks!
